library/src/main/java/com/devbrackets/android/exomedia/core/video/scale/MatrixManager.java

```diff
@@ -97,7 +97,9 @@ protected void applyCenterCrop(@NonNull TextureView view, @NonNull Matrix transf
         float yScale = (float)view.getHeight() / intrinsicVideoSize.y;
 
         float scale = Math.max(xScale, yScale);
-        applyScale(view, transformMatrix, scale, scale);
+        xScale = scale / xScale;
```

Comment:
Why are we dividing the calculate maxScale?  This will cause an inconsistent scaling (stretching)